### PR TITLE
Command to recursively import images from a given location

### PR DESCRIPTION
**Pull request by nightsh**
Syntax:

```
$ ./manage.py importimages [-n|--no-commit] <start_id> <collection_id> <path_to_import_from>
```

Example:

```
$ ./manage.py importimages 100 2 ../tbs-old/htdocs/RES/uploads/images
```

### CODE REVIEW

File: blacktail/management/commands/importimages.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+        self.width = dimensions[0]
+        self.height = dimensions[1]
+
+
+class Command(BaseCommand):
+    help = "Imports all images from a given path into a collection"
+
+    @transaction.atomic
+    def execute_sql(sql):
```
Comment (mgax) [inline blacktail/management/commands/importimages.py]: Does this work? Looks like a method, but there's no `self` argument. And it doesn't need to be part of the `Command` class anyway.

Comment (mgax) [reply blacktail/management/commands/importimages.py]: And you can be pretty sure that a single sql statement is atomic.

Comment (nightsh) [reply blacktail/management/commands/importimages.py]: Sorry, that was some last minute refactoring. It was a class method before, then I realized there's no need to stay in the same context... and I just removed any `self` mentions. I should've moved it outside the class declaration :slightly_smiling_face: 

Will patch it shortly.

Comment (nightsh) [reply blacktail/management/commands/importimages.py]: As for atomicity, yeah, it's redundant. I will remove it, when I first put it there I had no idea what will happen exactly in that function – good catch! :v:

**Comment by nightsh** [comment]
A dry run? Sure :)

**Comment by mgax** [comment]
@nightsh please fix the codacy review then merge it

**Pull request merged**